lib/seed_utils.rb

```diff
@@ -12,7 +47,7 @@ def seed_table_with_upload(klass, user, options = {})
     csv_name = "#{csv_type.underscore}.csv"
     csv_path = 'sample_csvs'
 
-    puts "Loading #{klass.name} from #{csv_path}/#{csv_name} ... "
+    load_table(klass, user, file_options, csv_name)
 
     uf = ActionDispatch::Http::UploadedFile.new(
```

Comment:
new method for creating the upload file object
`def upload_file(name, content_type)`